php_phongo.c

```diff
@@ -2375,8 +2375,19 @@ PHP_MINFO_FUNCTION(mongodb)
 	php_info_print_table_header(2, "mongodb support", "enabled");
 	php_info_print_table_row(2, "mongodb version", MONGODB_VERSION_S);
 	php_info_print_table_row(2, "mongodb stability", MONGODB_STABILITY_S);
+#ifdef HAVE_SYSTEM_LIBMONGOC
+	php_info_print_table_row(2, "libmongoc headers version", MONGOC_VERSION_S);
+	php_info_print_table_row(2, "libmongoc library version", mongoc_get_version());
+#else
```

Comment:
I would prefer to keep this consistent.
Otherwise we have to ask `what does phpinfo() report as "libmongoc headers version and "libmongoc library version" -- or -- libmongoc version?`.

Lets just always print out both compile time and lib